uint16 n_obj # as the maximum planning horizon
---
string msg
